geomstats/visualization/beta_distributions.py

```diff
@@ -0,0 +1,426 @@
+        plt.ylabel(r"$\beta$")
+
+    def plot_rendering(self, initial_point=[2, 2], size=[10, 10], sampling_period=1):
+        """Draws the beta manifold.
+
+        by Yiliang Chen & Allen Wang
+
+        Parameters
+        ----------
+        Initial_point : array-like, shape=[1, 2]
+            Defines initial point for plot rendering
+            Optional, default: [2,2]
+        size : array-like, shape=[..., 2]
+            Defines the range of the samples to be shown
+            Optional, default: [10,10]
+        sampling_period : float, >0
+            Defines the sampling period of the sampled data
+            Optional, default: 1
+        """
+        for value in initial_point:
+            if value < 0:
+                raise ValueError(
+                    "Initial Point {} is not in the first quadrant".format(
+                        initial_point
+                    )
+                )
+
+        sz = gs.array(size)
+        if sz.size != 2:
+            raise ValueError("size should be a 1*2 array")
+        x = gs.linspace(
+            initial_point[0],
+            (initial_point[0] + size[0] - 1) * sampling_period,
+            size[0],
+        )
+        y = gs.linspace(
+            initial_point[1],
+            (initial_point[1] + size[1] - 1) * sampling_period,
+            size[1],
+        )
+        points = [[i, j] for i in x for j in y]
+        points_x = [i[0] for i in points]
+        points_y = [i[1] for i in points]
+        fig = plt.figure(figsize=(5, 5))
+        ax = fig.add_subplot(111)
+
+        ax.scatter(points_x, points_y)
+        plt.xlabel(r"$\alpha$")
+        plt.ylabel(r"$\beta$")
+
+    def plot_grid(
+        self, size, initial_point=[2, 2], n_steps=100, n_points=10, step=1, **kwargs
+    ):
+        """Draws the grids of beta manifold.
+
+        by Yiliang Chen
+
+        Parameters
+        ----------
+        size : array-like, shape=[..., 2]
+            Defines the range of the grids to be shown.
+        initial_point : array-like, shape=[1,2]
+            Defines the initial point for plotting the beta manifold grid.
+            Optional, default: [2,2]
+        n_steps : int, >0
+            Defines the number of steps for integration.
+            Optional, default: 100
+        n_points : int, >0
+            Defines the number of points for interpolation.
+            Optional, default: 10
+        step : float, >0
+            Defines the length of a step for the grid
+            Optional, default: 1
+        """
+        for value in initial_point:
+            if value < 0:
+                raise ValueError(
+                    "Initial Point {} is not in the first quadrant".format(
+                        initial_point
+                    )
+                )
+
+        sz = gs.array(size)
+        if sz.size != 2:
+            raise ValueError("size should be a 1*2 array")
+        b = [(initial_point[0] + i * step) for i in range(size[0])]
```

Comment:
avoid using single letter for variable names: can you rename `b` to something that is more helpful for the reader?